src/yea_wandb/plugin.py

```diff
@@ -26,6 +26,21 @@ def fn_find(args, state):
     return None
 
 
+def fn_count_regex(args, state):
```

Comment:
add a trivial test that demonstrates how this is used in:
https://github.com/wandb/yea-wandb/blob/main/tests/plugin_test.py